.changelog/unreleased/summary.md

```diff
@@ -38,11 +38,16 @@ team, including:
     algorithm used by CometBFT for computing block times.
     When activated on a chain, it replaces the pre-existing BFT-time algorithm.
     See [spec](./spec/consensus/proposer-based-timestamp) doc for PBTS.
+8. Added support for [BLS12-381 curve](https://github.com/cometbft/cometbft/pull/2765) that can
```

Comment:
This sounds to me an experimental support, since we need a build flag.

Not sure if this is part of the major features.

Reply:
And if it is, we should add a link to what this "build tag" means.

Again, the release binaries would not contain it, so.... not a major feature.